samples/bookinfo/routing-rules/route-rule-all-v1.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: productpage-route
```

Comment:
Maybe just call it productpage? This is really an extension to `Service` definition, so it makes sense to match the names.

Reply:
Good point. Currently I'm using the convention `<service-name>-route` for `VirtualService` and `<service-name>-destination` for the corresponding `DestinationRule`. If I take your suggestion, what do you think would be a better name for the `DestinationRule`? I'm thinking maybe `<service-name>-rule` or `<service-name>-policy`.

Reply:
If the idea is to have a single destination rule per service, then it also makes sense to call them by the service name.

Reply:
Works for me. Done.